cf-java-logging-support-core/src/main/java/com/sap/hcp/cf/logging/common/converter/DefaultCustomFieldsConverter.java

```diff
@@ -33,6 +34,10 @@ public void setCustomFieldKeyNames(List<String> customFieldKeyNames) {
 		this.customFieldKeyNames = customFieldKeyNames;
 	}
 
+    private static class LoggerHolder {
```

Comment:
Did you apply the formatter on this file as well?

Reply:
Well, its hard to do it right. The formatter says 4 spaces, the file uses tabs. I followed the formatter as requested.
What I don't want to do is to reformat the whole file. This might be done in a separate PR.

Reply:
Ok, that means I formatted it the wrong way some time ago. 4 spaces is fine in that case. I am sure to touch this file myself again and will do the reformatting than. Sorry to have bothered you with that.